docs/workflow/requirements/windows-requirements.md

```diff
@@ -109,5 +109,4 @@ Simply edit the PATH variable's value and add the directory (with a semicolon se
 
 ### ARM64 (Experimental)
 
-* Install [CMake](https://cmake.org/download) using the `windows-i386` installer (`windows-x86_64` is not going to work, `C++ CMake tools for Windows` from VS might also not work).
-* Use MSBuild instead of Ninja (no official support for ARM64 yet).
+* Use MSBuild instead of Ninja (no official support for ARM64 yet) by passing `-msbuild` to either `build.cmd` or `build.ps1`.
```

Comment:
Is this needed when using tools that come with Visual Studio?

Ninja is installed as part of Visual Studio at 
`C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja`. Does this tools directory exist in Visual Studio on Arm64?

Reply:
Oh, there is a Ninja there and the build seems to find it just fine. I think we can just get rid of that then.